submissions/Chimoney-Python/chimoney/Base.py

```diff
@@ -84,10 +119,17 @@ def _handle_request(self, method_type, path, data=None, params=None):
             dict: The response from the Chi Money API.
         """
 
-        if data:
-            payload = json.dumps(data)
-        else:
-            payload = None
+        # if data:
+        #     payload = json.dumps(data)
```

Comment:
kindly remove redundant/commented out lines of code.

Reply:
Will do